Cpp2IL/Program.cs

```diff
@@ -504,9 +504,7 @@ public static int MainWithArgs(Cpp2IlRuntimeArgs runtimeArgs)
 
             //Pre-process processing layers, allowing them to stop others from running
             Logger.InfoNewline("Pre-processing processing layers...");
-            var layers = runtimeArgs.ProcessingLayersToRun.Clone();
-            RunProcessingLayers(runtimeArgs, processingLayer => processingLayer.PreProcess(Cpp2IlApi.CurrentAppContext, layers));
-            runtimeArgs.ProcessingLayersToRun = layers;
+            RunProcessingLayers(runtimeArgs, processingLayer => processingLayer.PreProcess(Cpp2IlApi.CurrentAppContext, runtimeArgs.ProcessingLayersToRun));
```

Comment:
This will cause a ConcurrentModificationException if a processing layer modifies the list of processing layers (e.g. to disable one it's incompatible with)